lib/client-assets.php

```diff
@@ -906,11 +906,12 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 	$allowed_block_types = apply_filters( 'allowed_block_types', true );
 
 	$editor_settings = array(
-		'alignWide'           => $align_wide || ! empty( $gutenberg_theme_support[0]['wide-images'] ), // Backcompat. Use `align-wide` outside of `gutenberg` array.
-		'availableTemplates'  => wp_get_theme()->get_page_templates( get_post( $post_to_edit['id'] ) ),
-		'blockTypes'          => $allowed_block_types,
-		'disableCustomColors' => get_theme_support( 'disable-custom-colors' ),
-		'titlePlaceholder'    => apply_filters( 'enter_title_here', __( 'Add title', 'gutenberg' ), $post ),
+		'alignWide'                => $align_wide || ! empty( $gutenberg_theme_support[0]['wide-images'] ), // Backcompat. Use `align-wide` outside of `gutenberg` array.
+		'availableTemplates'       => wp_get_theme()->get_page_templates( get_post( $post_to_edit['id'] ) ),
+		'blockTypes'               => $allowed_block_types,
+		'disableCustomColors'      => get_theme_support( 'disable-custom-colors' ),
+		'themeSupportsPostFormats' => current_theme_supports( 'post-formats' ),
```

Comment:
not sure the editor should be aware of the concept of "themes", should just rename the property something like `disablePostFormats` to match `disabledCustomColors`'s convention

Reply:
Code was updated 👍